src/include/daos/dtx.h

```diff
@@ -42,6 +42,23 @@ enum dtx_target_flags {
 	DTF_RDONLY			= (1 << 0),
 };
 
+enum dtx_grp_flags {
+	/* The group only contains read-only operations for the DTX. */
+	DGF_RDONLY			= (1 << 0),
+};
+
+enum dtx_mbs_flags {
+	/* The targets that are modified by the distributed transaction
+	 * are in the same single redundancy group.
+	 */
+	DMF_MODIFY_SRDG			= (1 << 0),
```

Comment:
I thought we discussed not choosing leader based on object class? i.e. dtx will store the leader information.

Reply:
sorry, what dos MBS stand for?

Reply:
This is only for single redundancy group based distributed transaction to be compatible with the original stand-alone modification.